Sources/MongoSwift/MongoClient.swift

```diff
@@ -45,6 +45,10 @@ public struct MongoClientOptions: CodingStrategyProvider {
     /// Determines whether the client should retry supported write operations (on by default).
     public var retryWrites: Bool?
 
+    /// Specifies how long the driver should attempt to select a server for before throwing an error. Defaults to 30
+    /// seconds (30000 ms).
+    public var serverSelectionTimeoutMS: Int?
```

Comment:
is this not one of the timeouts we were going to defer for a while? Or is bernie still thinking about it? I can't remember where we left of with that.

Reply:
it seems like we are safe to have. the likely outcome of the new timeout project is that this option will stick around but have a slightly different (and backwards compatible) new meaning